kadi/commands/tests/test_commands.py

```diff
@@ -624,7 +624,6 @@ def test_command_set_safe_mode():
     exp = """\
 2000:001:00:00:00.000 | COMMAND_SW       | ACPCSFSU   | CMD_EVT  | event=Safe_mode, event_date=2000:001:00:00:00, scs=0
 2000:001:00:00:00.000 | COMMAND_SW       | CSELFMT5   | CMD_EVT  | event=Safe_mode, event_date=2000:001:00:00:00, scs=0
-2000:001:00:00:00.000 | COMMAND_SW       | AONSMSAF   | CMD_EVT  | event=Safe_mode, event_date=2000:001:00:00:00, scs=0
```

Comment:
Command events no longer inserts a bogus NSM transition.

Reply:
That seems not a "support offset pitch" change?

Reply:
I updated the PR title and description to reflect the broader scope of this PR. Good catch.